lib/toSVG.js

```diff
@@ -52,7 +53,12 @@ module.exports = function(parsed) {
     if (!layerTable) {
       throw new Error('no layer table for layer:' + entity.layer);
     }
-    const rgb = colors[layerTable.colorNumber];
+
+    let rgb = colors[layerTable.colorNumber];
+    if(rgb == null) {
```

Comment:
1. should be ```if (``` to confirm to the rest of the codebase
1. should be ```===``` not ```==```, which requires ```=== undefined```